testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergCreateTable.java

```diff
@@ -85,4 +91,40 @@ public void testCreateTableAsSelect()
             onTrino().executeQuery("DROP TABLE " + tableName);
         }
     }
+
+    @Test(groups = {ICEBERG, STORAGE_FORMATS})
+    public void testCreateExternalTableWithInaccessibleSchemaLocation()
+    {
+        String schemaName = "schema_without_location";
+        String schemaLocation = "/tmp/" + schemaName;
+        hdfsClient.createDirectory(schemaLocation);
+
+        onTrino().executeQuery(format("CREATE SCHEMA iceberg.%s WITH (location = '%s')", schemaName, schemaLocation));
+
+        hdfsClient.delete(schemaLocation);
+
+        String tableName = "test_create_external";
+        String tableLocation = "/tmp/" + tableName;
+
+        String schemaAndTableName = format("iceberg.%s.%s", schemaName, tableName);
+        onTrino().executeQuery(format("CREATE TABLE %s (a bigint, b VARCHAR) WITH (location = '%s')", schemaAndTableName, tableLocation));
+
+        try {
+            onTrino().executeQuery("INSERT INTO " + schemaAndTableName + "(a, b) VALUES " +
+                    "(NULL, NULL), " +
+                    "(-42, 'abc'), " +
+                    "(9223372036854775807, 'abcdefghijklmnopqrstuvwxyz')");
+            assertThat(onTrino().executeQuery("SELECT * FROM " + schemaAndTableName))
+                    .containsOnly(
+                            row(null, null),
+                            row(-42, "abc"),
+                            row(9223372036854775807L, "abcdefghijklmnopqrstuvwxyz"));
+        }
+        catch (Exception ex) {
+            throw ex;
+        }
```

Comment:
redundant, a left over?

Reply:
I added it back b/c it wouldn't compile without the catch. We could remove the `try` completely?

Reply:
Yes, please remove the `try` completely.

Reply:
done